console/src/main/scripts/plugins/accessPortal/vendor/js/custom-cases.js

```diff
@@ -1,10 +1,12 @@
+var attachmentsRequest = 'hawkular-redhat-access-integration-backend/attachments';
+
```

Comment:
That's because those JavaScripts are not coming from us *or* are very temporary, so, it's a sign that we should throw it away once a proper solution is offered by the Red Hat Access folks. This code is 90% the one that exists for RHQ.